components/Collections/src/private/MDCCollectionViewStyler.m

```diff
@@ -246,6 +246,35 @@ - (void)setCellStyle:(MDCCollectionViewCellStyle)cellStyle {
   _cellStyle = cellStyle;
 }
 
+- (BOOL)shouldHideSeparatorForCellLayoutAttributes:(MDCCollectionViewLayoutAttributes *)attr {
+  BOOL shouldHideSeparator = self.shouldHideSeparators;
+  if (!self.delegate)
+    return shouldHideSeparator;
```

Comment:
Please add curly braces  (`{}`) to surround the expression following the conditional (`if`)